src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -79,10 +79,10 @@ def load_arguments(self, _):
                    options_list=['--assign-endpoint', c.deprecate(target='--is-public', redirect='--assign-endpoint', hide=True)])
         c.argument('assign_identity', arg_type=get_three_state_flag(),
                    help='If true, assign managed service identity.')
-        c.argument('cpu', type=int, default=1,
-                   help='Number of virtual cpu cores per instance.')
-        c.argument('memory', type=int, default=1,
-                   help='Number of GB of memory per instance.')
+        c.argument('cpu', type=str, default="1",
+                   help='CPU resource quantity. Should be 500m or number of CPU cores.')
+        c.argument('memory', type=str, default="1Gi",
+                   help='Memory resource quantity. Should be 512Mi or #Gi, e.g., 1Gi, 3Gi.')
```

Comment:
May I ask this is a breaking change for users, right?

Reply:
The shell environment does not distinguish int or str arguments. The following commands works both in the previous version and the version being reviewed:

```
az spring-cloud app update -n test-app --cpu 1 --memory 1
```

The update should be a super set of the existing functionality. Is this still considered as breaking change?

Reply:
OK, I think it should be OK~